src/CoreWCF.WebHttp/src/CoreWCF/Configuration/ServiceModelWebServiceBuilderExtensions.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using CoreWCF.Description;
+using CoreWCF.OpenApi.Middleware;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace CoreWCF.Configuration
+{
+    public static class ServiceModelWebServiceBuilderExtensions
+    {
+        public static IServiceBuilder AddServiceWebEndpoint<TService, TContract>(
+            this IServiceBuilder builder,
+            IApplicationBuilder app,
+            WebHttpBinding binding,
+            string address,
```

Comment:
I think there might be some benefit to having an overload which doesn't take a WebHttpBinding and have the extension method instantiate it.

Reply:
This has been addressed.